features/informatievragen-fields.feature

```diff
@@ -0,0 +1,621 @@
+      | naam                              | waarde                |
+      | verblijfadres.straat              | <straat>              |
+      | verblijfadres.huisnummer          | <huisnummer>          |
+      | verblijfadres.postcode            | <postcode>            |
+      | verblijfadres.locatiebeschrijving | <locatiebeschrijving> |
+      | verblijfadres.regel1              | <regel1>              |
+
+      Voorbeelden:
+      | burgerservicenummer | gemeenteVanInschrijving | gemeente omschrijving              | straat    | huisnummer | postcode | woonplaats | locatiebeschrijving        | land | land omschrijving | regel1            | regel2       | regel 3       |
+      | 999993653           | 0599                    | Rotterdam                          | Boterdiep | 31         | 3077AW   | Rotterdam  |                            |      |                   |                   |              |               |
+      | 000009921           | 0518                    | 's-Gravenhage                      |           |            |          |            | Woonboot in de Grote Sloot |      |                   | Locatie           |              |               |
+      | 999995121           | 1999                    | Registratie Niet Ingezetenen (RNI) |           |            |          |            |                            | 5002 | Frankrijk         | 14 Rue Camulogene | 75015 Parijs |               |
+      | 999993483           | 1999                    | Registratie Niet Ingezetenen (RNI) |           |            |          |            |                            | 5010 | België            | Rue du pomme 25   | Bruxelles    | postcode 1000 |
+      | 999993586           |                         |                                    |           |            |          |            |                            | 0000 | Onbekend          |                   |              |               |
+
+    @gba
+    Abstract Scenario: met fields is gevraagd om adresregel2
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                                 | waarde                    |
+      | burgerservicenummer                  | <burgerservicenummer>     |
+      | gemeente van inschrijving (09.10)    | <gemeenteVanInschrijving> |
+      | gemeenteVanInschrijving.omschrijving | <gemeente omschrijving>   |
+      En de persoon heeft de volgende 'verblijfplaats' gegevens
+      | naam                             | waarde                |
+      | straatnaam (11.10)               | <straat>              |
+      | naam openbare ruimte (11.15)     | <straat>              |
+      | huisnummer (11.20)               | <huisnummer>          |
+      | postcode (11.60)                 | <postcode>            |
+      | woonplaats (11.70)               | <woonplaats>          |
+      | locatiebeschrijving (12.10)      | <locatiebeschrijving> |
+      | land.code                        | <land>                |
```

Comment:
Uit de regel 400 t/m 402 maak ik niet op dat de velden land.code en `land.omschrijving' van belang zijn voor dit scenario. Ik zou ze daarom hier verwijderen. Ze vertroebelen alleen maar het scenario.
Verwijder ze dan ook uit de tabel van regel 439 t/m 444.

Je kunt je ook afvragen waarom je in dit scenario de velden straat, aanduidingBijHuisnummer, huisnummer, huisletter, huisnummertoevoeging, locatiebeschrijving, regel 1 adres buitenland en regel 3 adres buitenland hier en in de tabel zou opnemen.

Reply:
zonder land en de andere adresvelden is het geen consistente data (niet conform LO en HUP). 
met het opnemen van de overige velden toont het scenario op deze manier dat bij vragen adresregel2 alleen gegevens belangrijk voor adresregel2 worden geleverd, geen gegevens die belangrijk zijn voor adresregel1, adresregel3 of adressering.land